### DISCUSSION TITLE
为什么不能多选节点了？

### DISCUSSION THREAD

**Discussion by lixiaoyu0123** [v2rayA/v2rayA#1481] [General]
新版本不能多选节点了，负载均衡怎么办呢？只能通过配置多个出站来解决吗？比起之前能多选要麻烦了一些。
为什么去掉多选的方式呢？

**Comment by lixiaoyu0123** [comment]
为什么amd64的版本的容器就不能选择多节点，arm64版本的容器却可以选择多节点呢？

**Reply by gdtv** [reply]
我刚好相反，几个月前安装的amd64版本可以选择多节点，刚刚新安装的arm64版本不能选择多节点了。

**Comment by txhtxh11** [comment]
同问

**Comment by lck1115** [comment]
同求负载均衡

**Comment by keinoki** [comment]
不能多节点的看下自己用的是 xray 还是 v2ray，从目前的逻辑上来看 v2ray 可以多选，xray 不能多选

**Reply by YoungxuJi** [reply]
怎么看是xray还是v2ray?我用的是docker 2.2.6.6版本

**Reply by keinoki** [reply]
看启动日志，应该有个 V2Ray binary is xxx

**Comment by lovefan-fan** [comment]
V2RAYA_V2RAY_BIN 的值应当是 /usr/local/bin/v2ray 或 /usr/local/bin/xray，默认的核心是 xray

**Reply by kingfxx** [reply]
谢谢,这个试了确实有用

**Comment by biutv** [comment]
怎么选择安装v2ray，而不是xray呢？？docker pull mzz2017/v2raya直接这样装的··目前不能多选。。还有个问题，万一选的节点有问题他会自动切换其他延迟低的节点吗？

**Reply by milkpodd** [reply]
用V2RAYA_V2RAY_BIN环境变量指定用xray还是v2ray核心的路径

**Comment by github-office** [comment]
Socks5 是不是也不能设置密码了？（xray核心）

**Comment by Satan023** [comment]
https://github.com/v2rayA/v2raya-openwrt/blob/master/README.zh-cn.md
选择一个你喜欢的内核，v2ray 或 Xray
如果两个内核被同时安装，默认使用后者
维护者注：由于目前 v2ray 的透明代理支持欠佳，因此更推荐使用 Xray
opkg install xray-core
